src/protojure/pedestal/core.clj

```diff
@@ -176,7 +176,7 @@
     (write-streaming-data ch is)))
 (defmethod transmit-body :default
```

Comment:
I seem to recall an earlier iteration of this work had handlers for both raw bytes and :default which had identical handlers, so they were collapsed into :default.  I'm thinking that with this change, you'd also need to add a handler for bytes again, and just plumb them straight through.